internal/credit/balance_snapshot/balance.go

```diff
@@ -0,0 +1,73 @@
+package balancesnapshot
```

Comment:
`balance_snapshot` vs `balancesnapshot`

Keeping package and folder name in sync is preferred.